images/kubectl-build-deploy-dind/scripts/exec-generate-sbom-configmap.sh

```diff
@@ -27,31 +33,23 @@ if [ -f "${SBOM_OUTPUT_FILE}" ]; then
             --from-file=${SBOM_OUTPUT_FILE} \
             -o json \
             --dry-run=client | kubectl replace -f -
-        kubectl --insecure-skip-tls-verify \
-            -n ${NAMESPACE} \
-            label configmap ${SBOM_CONFIGMAP} \
-            lagoon.sh/insightsType=sbom \
-            lagoon.sh/buildName=${LAGOON_BUILD_NAME} \
-            lagoon.sh/project=${PROJECT} \
-            lagoon.sh/environment=${ENVIRONMENT} \
-            lagoon.sh/service=${IMAGE_NAME}
-
     else
         # Create configmap and add label (#have to add label separately: https://github.com/kubernetes/kubernetes/issues/60295)
         kubectl --insecure-skip-tls-verify \
             -n ${NAMESPACE} \
             create configmap ${SBOM_CONFIGMAP} \
             --from-file=${SBOM_OUTPUT_FILE}
-        kubectl --insecure-skip-tls-verify \
-            -n ${NAMESPACE} \
-            label configmap ${SBOM_CONFIGMAP} \
-            lagoon.sh/insightsType=sbom \
-            lagoon.sh/buildName=${LAGOON_BUILD_NAME} \
-            lagoon.sh/project=${PROJECT} \
-            lagoon.sh/environment=${ENVIRONMENT} \
-            lagoon.sh/service=${IMAGE_NAME}
     fi
+    kubectl --insecure-skip-tls-verify \
+        -n ${NAMESPACE} \
+        label configmap ${SBOM_CONFIGMAP} \
+        lagoon.sh/insightsType=sbom-gz \
```

Comment:
@bomoko or @timclifford figured to change the label here, so that the receiving end knows it's getting a gzipped sbom. Does this change any processing?